### ISSUE TITLE
Control Panel Backup

### ISSUE THREAD

**Issue by mastipgithub** [ONLYOFFICE/CommunityServer#430]
Portal Version: 
![image](https://user-images.githubusercontent.com/4679142/197183735-8ab79abe-f405-4b40-91d1-de62526e4cf7.png)

After  portal restore from control panel I have few problems!

1. I can't open my old files in documents server but i can create new files so document server works ok. Even if i copy files to folder there is still a problem.

![image](https://user-images.githubusercontent.com/4679142/197184379-2eb8efb6-cee2-4ca1-8e1b-e0d5a75c286f.png)

I checked mounted folder / app / onlyoffice / CommunityServer / data / Products / Files / 00/00/01 / folder_1000 a was only few folder file_1, file_2, file_3. In ma backup archive there many folders, last folder is 311.

2. I don't see any icons from custom navigation links

![image](https://user-images.githubusercontent.com/4679142/197184634-568a9cde-66ff-4748-8b2f-c0020435c7ac.png)

I checked mounted folder / app / onlyoffice / CommunityServer / data / Studio / 00/00/01 / and there was no customnavigation folder!

3. In Control Panel there are no files in common folder in documents module. 

Before portal restore there was no problems. I installed clean version, everything was working fine. Then I sent backup file to document module and I restored portal from this file.
4. These are logs after portal restore.

[Community-Server_logs.zip](https://github.com/ONLYOFFICE/CommunityServer/files/9838690/Community-Server_logs.zip)

Widzę błędy dotyczące dostępu do plików, błąd bazy danych z powodu serwera e-mail, którego nie zainstalowałem.

**Comment by Carazyda** [comment]
Hello @mastipgithub Was a mail server installed on the server from which you took the backup? Is the mail server installed on the new server where you make the restore?

**Comment by mastipgithub** [comment]
I lost my virtual machine with my last server, but in backup folder there is a folder with mail server settings. So I have to install new portal with mail server and then after restore will be ok? How this can make a problem with file access? Maybe I had mail server but I did not use it.

**Comment by Carazyda** [comment]
Yes, we have a bug with this (59475 in our private bug tracker). Please install a new server with a mail server and try running a restore.
